activemq-broker/src/main/java/org/apache/activemq/broker/jmx/BrokerViewMBean.java

```diff
@@ -179,9 +179,17 @@ public interface BrokerViewMBean extends Service {
     @MBeanInfo("Topics (broadcasted 'queues'); generally system information.")
     ObjectName[] getTopics();
 
+    @MBeanInfo("Total number of topics")
+    int getTotalTopicsCount();
+
+
     @MBeanInfo("Standard Queues containing AIE messages.")
     ObjectName[] getQueues();
 
+    @MBeanInfo("Total number of queues")
+    int getTotalQueuesCount();
+
+
```

Comment:
This should add getTotalTempQueueCount and getTotalTempTopicCount while we are here

Reply:
Can you elaborate ?

Reply:
There are 4 total sets of destinations -- Queues, Topics, TempQueues and TempTopics. If we are adding the metric for TotalQueueCount and TotalTopicCount, we should also add TotalTempQueueCount and TotalTempTopicCount.

Reply:
Oh you mean adding `getTotalTempQueueCount()` and `getTotalTempTopicCount()`. Yeah, agree, it makes sense.

Reply:
Btw, this is already resolved :)